uw-frame-components/portal/settings/partials/settings.html

```diff
@@ -10,16 +10,19 @@
           <uw-setting-option></uw-setting-option>
         </div>
       </div>
-      <!-- Theme selector and 'danger zone' -->
+      <!-- Theme selector and "danger zone" -->
       <div layout="column" flex-gt-sm="40">
-        <h4 class="md-subhead">Theme Selector<br><small>Pick which place you call home.</small></h4>
-        <select class="form-control" ng-options="item as item.name for item in THEMES track by item.name" ng-model="portal.theme">
-        </select>
+        <h4 class="md-subhead">Theme Selector</h4>
+        <md-input-container>
+          <md-select ng-model="portal.theme" placeholder="Pick which place you call home" aria-label="Select a theme">
```

Comment:
I agree this behaves the same as in the [upstream demo](https://material.angularjs.org/latest/demo/select) and I agree it's not a dealbreaker for moving forward. 

I still say it feels bugged.  Clicking through many angular-material open issues about `select` I didn't see precisely this issue, though I saw a couple conceptually close to it. Anyway, I'm fine with our strategy of adopting widely-adopted technology so that if this is a real issue, well, it'll annoy lots of adopters and someone will fix it. :smile: